rero_ils/query.py

```diff
@@ -135,14 +153,25 @@ def search_factory(self, search, query_parser=None):
     """
     def _default_parser(qstr=None, query_boosting=[]):
         """Default parser that uses the Q() from elasticsearch_dsl."""
+        query_type = 'query_string'
+        default_operator = 'OR'
+        if request.args.get('simple'):
```

Comment:
Why operator is set from *simple* argument. I think it's not fully related and should be passed as another parameter.

Reply:
I do not want to have two parameters even if you are again right, it is cleaner.